samples/application-mobility/csm_application_mobility_v030.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       - name: application-mobility-controller-manager
         # enable: Enable/Disable application mobility controller-manager
         enabled: true
-        image: dellemc/csm-application-mobility-controller:v0.3.0
+        image: amaas-eos-mw1.cec.lab.emc.com:5098/csm-application-mobility:latest
```

Comment:
Internal image name shouldn't be in repo

Reply:
changed it